src/Processors/Formats/Impl/ParquetBlockInputFormat.cpp

```diff
@@ -54,6 +54,18 @@ namespace ErrorCodes
         }                                                              \
     } while (false)
 
+static Field createDecimalField(const Int256 & value, UInt64 precision_value, UInt64 scale)
```

Comment:
precision_value -> precision is better

Reply:
why precision type and scale type is UInt64 ?

Reply:
just refer to  `createDecimal`
```cpp
template <template <typename> typename DecimalType>
inline DataTypePtr createDecimal(UInt64 precision_value, UInt64 scale_value)
```

Reply:
fix it.